CONTRIBUTING.md

```diff
@@ -47,7 +47,31 @@ and let us know if it's not up-to-date (even better, submit a PR with your  corr
 
 ## Getting Started
 
-For day-to-day development and normal contributions, the following SDKs and tools are required:
+### Gitpod
+
+For setting up a local development environment,
+we recommend using [Gitpod](http://gitpod.io) -
+a service that allows you to spin up an in-browser
+Visual Studio Code-compatible editor,
+with everything set up and ready to go for CDK development.
+Just click the button below to create your private workspace:
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/aws/aws-cdk)
+
+This will start a new Gitpod workspace,
+and immediately kick off a build of the CDK code.
+Once it's done (it takes around an hour, unfortunately),
+you can work on any package that you want to modify,
+as described in ['Quick Iteration'](#quick-iteration) below.
```

Comment:
I think the quick iteration section needs a refresh. It still references `lr watch` and `lr test`
Are those still accurate / representative of the typical flow. Asking as I've never actually used either...

Reply:
@rix0rrr so I followed up and it does work and the section is still relevant.

It *does* require [useful aliases](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#useful-aliases) as a pre-requisite to jumping into quick iteration. I still think that's worth mentioning here.

I won't hold up the PR if that doesn't bug you though